src/User/Command/EditUserHandler.php

```diff
@@ -58,15 +58,14 @@ public function handle(EditUser $command)
 
         $user = $this->users->findOrFail($command->userId, $actor);
 
-        $canEdit = $actor->can('edit', $user);
         $isSelf = $actor->id === $user->id;
 
         $attributes = Arr::get($data, 'attributes', []);
         $relationships = Arr::get($data, 'relationships', []);
         $validate = [];
 
         if (isset($attributes['username'])) {
-            $actor->assertPermission($canEdit);
+            $actor->assertPermission($actor->can('user.editUsername'));
```

Comment:
Same applies here; this should apply on the $user object, and the `'user'` prefix is unnecessary

Reply:
The user prefix is required, in my testing without the user. prefix it did not work correctly and always returned false. This is one of those cases of the magic not working correctly.